src/lib.rs

```diff
@@ -123,6 +123,8 @@ pub use dataset::{
     LayerOptions, Transaction,
 };
 pub use driver::{Driver, DriverManager};
+#[cfg(all(major_is_3, minor_ge_6))]
```

Comment:
We probably want `ge`, not `is` on these.

Reply:
yes something like `#[cfg(any(major_ge_4, all(major_is_3, minor_ge_6)))]`